disco_aws_automation/disco_elb.py

```diff
@@ -168,21 +168,22 @@ def get_or_create_target_group(self, group_name, vpc_id=None,
                 port = 80
                 health_protocol = 'HTTP'
                 health_port = '80'
-            if health_check_path:
-                health_check_path = health_check_path
-            else:
-                health_check_path = '/'
+
+            health_check_path = health_check_path or '/'
+
+            health_args = dict()
+            health_args['HealthCheckProtocol'] = health_protocol
+            if health_protocol in ('HTTP', 'HTTPS'):
+                health_args['HealthCheckPort'] = health_port
+                health_args['HealthCheckPath'] = health_check_path
```

Comment:
These values will also default if not set in port config to the instances port and '/'

Reply:
This is done by aws and wont be set if the protocol type doesnt support it